kong/db/strategies/postgres/connector.lua

```diff
@@ -315,8 +315,7 @@ end
 
 
 function _mt:init_worker(strategies)
-  if ngx.worker.id() == 0 then
-
+  if ngx.worker.id() == 0 and #kong.configuration.admin_listeners > 0 then
```

Comment:
Emmm, should this not instead check if the node doesn't have proxy listeners?
Thinking traditional mode.

cc @bungle @hanshuebner